TextMining.py

```diff
@@ -0,0 +1,125 @@
, '‘VERY': 1, 'thing,': 4, 'advisable--“’': 1, 'about,’': 1, 'INDEMNITY': 1, 'queerest': 1, 'March': 31, 'now--but': 1, 'beloved': 1, '‘But,': 1, '‘Then,': 1, 'comfort,': 1, 'Mississippi': 1, 'hippopotamus,': 1, 'hastily.': 4, 'players,': 2, 'additions': 1, 'sense': 1, '‘Dinah’s': 1, 'included': 2, 'story.’': 1, 'fish)--and': 1, 'Rabbit’s': 3, 'positively': 1, 'splashed': 1, 'Miss,': 2, 'contemptuous': 1, 'flurry': 1, '“There’s': 1, 'throughout': 1, 'confirmation': 1, 'afore': 1, 'except': 4, 'talking:': 1, 'court.': 3, 'first.’': 1, 'stingy': 1, 'MINE.’': 1, 'commotion': 1, 'indeed': 3, 'denying': 1, 'end.’': 1, 'NEGLIGENCE,': 1, 'seen--everything': 1, 'Director': 1, 'fluttered': 1, 'talking.': 2, 'yards': 1, 'Dormouse!': 1, 'bats,': 1, 'raised': 2, 'arm-in-arm': 1, 'about;': 1, 'promotion': 1, 'raising': 1, 'entrance': 1, 'cheeks,': 1, 'fairy-tales,': 1, 'raw.’': 1, 'foolish': 1, 'timidly;': 1, 'particularly': 1, 'won’t,': 1, 'tried.': 1, 'circle,': 1, 'derived': 1, '‘--as': 1, 'insolence': 1, 'speaking,': 3, 'cucumber-frames': 1, 'carried': 4, 'close,': 1, 'displaying,': 2, 'IMPLIED,': 1, 'likes.’': 1, 'unhappy.': 1, 'rising': 1, 'fishes': 1, 'bleeds;': 1, 'whispers': 1, 'sharply;': 1, '‘--you': 1, 'hers--she': 1, 'themselves.”’': 1, 'politely,': 3, 'Off--’': 1, 'sadly:--': 1, 'meant,’': 1, 'ACTUAL,': 1, 'grown': 7, 'Rabbit:': 1, 'sides': 4, 'remarked;': 1, 'hatter.’': 1, 'rightly': 1, 'adventures': 2, 'playing': 2, 'life.': 3, 'home?’': 1, 'something!’': 1, 'pleased,': 1, 'cup,’': 1, 'directions,': 2, 'timidly.': 1, 'critical': 1, 'cat,’': 1, 'angrily': 2, '“Such': 1, 'mouse--a': 1, 'green,': 1, 'bathing': 1, 'FOOT,': 1, 'savage!’': 1, 'plates': 1, 'with?’': 1, 'distance,': 4, 'nervous,': 1, 'bring': 2, 'jury.': 2, 'offended!’': 1, 'replace': 1, 'more.’': 1, 'whenever': 1, 'Cat,’': 1, 'dispute': 2, 'spoke': 4, 'twinkling': 3, 'stop.’': 1, 'balls': 1, 'them.': 2, 'verse,’': 2, 'given': 2, 'violently': 2, '‘Stolen!’': 1, 'rate,': 2, 'straightening': 1, 'say.’': 2, 'happened.': 1, 'circumstances.': 1, 'called': 15, 'court!’': 1, 'tea--’': 1, 'thought:': 1, 'relief.': 2, 'sleep,': 1, 'Creating': 1, 'thunder,': 1, 'listened,': 1, 'obtaining': 2, 'identify,': 1, 'middle,': 3, 'back-somersault': 1, 'sharply.': 1, 'that;': 1, 'lessons!’': 1, 'included.': 1, 'holding': 2, 'brother’s': 1, 'Mission': 1, 'now,’': 4, 'minded': 1, 'taken': 4, 'dog’s': 1, 'boldly:': 1, 'somebody,': 1, '‘unless': 1, 'baked': 1, '‘Only': 2, 'wrong!’': 1, 'moment': 21, 'knew,': 1, 'indemnify': 1, 'little,’': 1, 'Sections': 1, 'replied': 14, 'beast,': 1, '“Defects,”': 1, 'nibbled': 2, 'giddy.’': 2, 'trusts': 1, '“Project': 5, 'directly,': 1, 'uncorked': 1, 'teases.’': 1, 'nonproprietary': 1, 'other:': 3, 'Title:': 1, 'Dormouse;': 2, 'extraordinary': 2, '‘advance': 1, 'high,': 3, '‘--Mystery,': 1, 'rustling': 1, 'White': 22, 'adventures--beginning': 1, '11-0.txt': 1, 'wearily.': 1, 'child': 3, '‘arrum.’)': 1, 'exempt': 2, 'unenforceability': 1, 'which': 44, 'pope,': 1, 'can--’':
```

Comment:
Usually, this part of code `if __name__ == "__main__":` goes at the bottom of your file.